server/app/services/cloud/aws/SimpleEmail.java

```diff
@@ -73,6 +74,9 @@ public void send(ImmutableList<String> toAddresses, String subject, String bodyT
 
       SendEmailRequest emailRequest =
           SendEmailRequest.builder().destination(destination).message(msg).source(sender).build();
+      if (logger.isDebugEnabled()) {
+        logger.debug("Sending email:\n{}", emailRequest);
```

Comment:
Definitely correct me if I'm wrong here. Based on my understanding, these logs will show up for all providers. Presumably the toString() of the email request object would contain both the payload of the email as well as the email address. If that's the case, wouldn't these be considered PII and not something we'd want to log?

Perhaps the answer is just that "we'll never turn on debugging logging for prod", but I'd be a lot more comfortable if we instead logged only the application ID or something that didn't contain an email address.

Relatedly, I was poking around with @nbeloglazov yesterday, and we found that localstack has a debug endpoint for sent emails (https://docs.localstack.cloud/aws/ses/). We could presumably use this to assert email-related code for bin/run-dev and browser tests.

Reply:
yes and yes.  (adding an example in the desc too)

At the end of the day the system and the people running it could log everything if they desired.  The defaults in the logging conf already hide DEBUG logging that would expose PII otherwise, one in particular logs all SQL queries and results which would include emails.  So this seems quite in line with that.  It's also completely possible there's already a deeper logging target that we could set to debug to see this content. 

Adding the endpoint seems great, but also seemingly a larger investment than we have prioritized now, and likewise only works appropriately if correctly configured and used in dev, which is the same as this PR.

Feel free to have bion make a call.

Reply:
Also the problem with not logging the content is we can never dev test new email content.  So eitherway we need to be able to see the full payload locally.

Reply:
Actually I'll just add @bion  :D

Reply:
That rationale all makes sense. I just wanted to be purposeful about when we wanted to add logging such as this. SQL logging at debug level definitely seems concerning for the same reasons to me, but that's probably part of a larger cleanup.

I think the concern from my perspective is that the person viewing logs doesn't necessarily have to be the same as the person "running the system".

Regarding adding the endpoint and prioritization, I perhaps was being a bit too vague. The endpoint already exists and is exposed by localstack. On a run-dev environment, I "logged" in as f@example.com and submitted an application. I then navigated to http://localstack:4566/_localstack/ses and saw a JSON representation of the email that was sent. IMO, this is a stronger test that the email was sent than a log statement prior to calling the endpoint. Totally fine to defer this, I just wanted to call out that there's no new implementation required to have localstack debugging.

Reply:
Reverting as we determined offline that all email content is already logged to http://localstack:4566/_localstack/ses (note you need to setup a /etc/hosts for that to work)